Python Tutorial Exercises(Week 5)/#4.py

```diff
@@ -0,0 +1,26 @@
+'''
+use if len(i) >= 8 to find all the strings that have at least 8 characters or more.
+    characters = [char for char in andrews_passwords if len(char) >= 8]
+
+use if not sym.isalnum() to get the strings with least one character is not alphanumeric
+    symbol = [sym for sym in andrews_passwords if not sym.isalnum()]
+
+use if sum(map(str.islower, low)) >= 1 to get the strings with least one character is lowercase letter
+    lowercase  = [low for low in andrews_passwords if sum(map(str.islower, low)) >= 1]
+
+use if sum(map(str.isupper, up)) >= 1 to get the strings with least one character is uppercase letter
+    uppercase  = [up for up in andrews_passwords if sum(map(str.isupper,up)) >= 1]
+    
+use if sum(map(str.isdigit, num)) >= 2 to get the strings with least two character is numbers
+    number = [num for num in andrews_passwords if sum(map(str.isdigit, num)) >= 2 ]
+
+'''
+
+andrews_passwords = ["123456", "hAhA123!", "MyCarR0cks*", "EasyPassword123$$", "AnotherPassword??", "letmein"]
+
+def check_password(andrews_passwords):
+
+    final = [num for num in [up for up in [low for low in [sym for sym in [char for char in andrews_passwords if len(char) >= 8] if not sym.isalnum()] if sum(map(str.islower, low)) >= 1] if sum(map(str.isupper,up)) >= 1] if sum(map(str.isdigit, num)) >= 2 ]
```

Comment:
there seems to be a misunderstanding for why this function is needed, take the below for example:
```
def check_even(number):
    return (number % 2 == 0)

numbers = [0, 3, 76, 523, 64, 6234]
even_numbers = [n for n in numbers if check_even(n)]
print(even_numbers)
```